platform/ios/scripts/script_resources/MapboxDemo/MapboxDemo.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don’t think this `xcworkspace` is necessary to commit — because this project is so simple, it doesn’t contain anything Xcode can’t automatically recreate. [See this StackOverflow question.](https://stackoverflow.com/q/10956312/2094275).

Reply:
Removed https://github.com/mapbox/mapbox-gl-native/pull/10524/commits/424a287255e5764e1180c14fdffaaaff68d8ae26